src/sdk/sdk.test.ts

```diff
@@ -253,21 +259,21 @@ describe("sanitizeResourceName", () => {
   it("should replace sequences of non-alphanumeric characters with a single -", () => {
     const resourceName = "test-*^%- -!=!resource";
     const sanitizedResourceName = sanitizeResourceName(resourceName);
-    expect(sanitizedResourceName).toEqual("test-resource");
+    expect(sanitizedResourceName).toEqual("test------resource");
   });
 
-  it("should truncate name to 250 characters", () => {
+  it("should truncate name to 253 characters", () => {
     const resourceName =
-      "test-resourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresource";
+      "test-resourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourcemoreletters";
     const sanitizedResourceName = sanitizeResourceName(resourceName);
     expect(sanitizedResourceName).toEqual(
-      "test-resourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresourceresou",
+      "test-resourceresourceresourceresourceresourceresourceresourcere",
     );
   });
 
   it("should remove leading and trailing non-letter characters", () => {
```

Comment:
```suggestion
  it("should remove leading and trailing non-alphanumeric characters", () => {
```
I think this test name better reflects the assertion